src/model.jl

```diff
@@ -53,12 +54,12 @@ function PowerManagementProblem(fq, fl, d, pmax, gmax, A, B; τ=TAU)
         p <= pmax,
         -g <= 0, 
         g <= gmax,
-        0 == A*p - B*g + d,
+        0 == A*p - B*g + d + ds,
     ])
 
     params = (fq=fq, fl=fl, d=d, pmax=pmax, gmax=gmax, A=A, B=B, τ=τ)
 
-    return PowerManagementProblem(problem, p, g, params)
+    return PowerManagementProblem(problem, p, g, nothing, params)
 end
 
 PowerManagementProblem(net::PowerNetwork, d) =
```

Comment:
Does this need to be changed too?

Reply:
You're right!

Reply:
well, I guess it's not necessary per se. I just thought that having only one `struct` for individual subproblems would be nice, and therefore I repurposed the `PowerManagementProblem` one to allow for this flexibility. If you don't like it, I can find another way